collectors/osv/tests/test_collectors.py

```diff
@@ -14,7 +13,6 @@
 
 
 class TestOSVCollector:
-    @freeze_time(datetime(2020, 12, 12))  # freeze against top of the second crossing
```

Comment:
We have more usages of `@freeze_time` in our tests (e.g. also in CVEorg collector), but these can probably wait.